server/form.php

```diff
@@ -27,7 +37,7 @@
 	} else if ($retrieve == "options") {
 		$questionId = $_REQUEST['questionId'];
 
-		$optionsStatement = $conn->prepare('SELECT answerId, string, archived FROM Answer
+		$optionsStatement = $conn->prepare('SELECT possibleAnswerId, string, archived FROM PossibleAnswer
 			WHERE questionId = ' . $questionId);
```

Comment:
Uh isn't this bad... You're using a PreparedStatement but then just appending the variable part. You really should use the execute(array(variables)) method I think, as to prevent SQL injection